### DISCUSSION TITLE
In search of more exploits

### DISCUSSION THREAD

**Discussion by Sxri** [3kh0/ext-remover#203] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello, I am pretty good in coding, especially javascript. However, I can't seem to find many ways to exploit stuff on my computer. I've made a few exploits for my chromebook and friends chromebooks, involving lagging the teachers computer when they look at our screen through the extension, unblock all sites (patched now), and some other random ones. but I'm looking for some more considering I can't really find much these days or make any that won't work

I'm searching for suggestions or ideas that may work. If you have any spare exploits please, pass me them to try!

Thanks!

Sincerely, lurk/sxri.

**Comment by ghost** [comment, marked as answer]
Have you seen mine? Also, try an adblocker bookmarklet, which may work for overlay blockers. `javascript:(function(){   var selectors = [    /* By ID: */    '#sidebar-wrap',%20'#advert',%20'#xrail',%20'#middle-article-advert-container',%20%20%22adinplay%22,%20%20'#sponsored-recommendations',%20'#around-the-web',%20'#sponsored-recommendations',%20%20%20%20'#taboola-content',%20'#taboola-below-taboola-native-thumbnails',%20'#inarticle_wrapper_div',%20%20%20%20'#rc-row-container',%20'#ads',%20'#at-share-dock',%20'#at4-share',%20'#at4-follow',%20'#right-ads-rail',%20%20%20%20'div#ad-interstitial',%20'div#advert-article',%20'div#ac-lre-player-ph',%20%20%20%20/*%20By%20Class:%20*/%20%20%20%20'.ad',%20'.avert',%20'.avert__wrapper',%20'.middle-banner-ad',%20'.advertisement',%20%20%20%20'.GoogleActiveViewClass',%20'.advert',%20'.cns-ads-stage',%20'.teads-inread',%20'.ad-banner',%20%20%20%20'.ad-anchored',%20'.js_shelf_ads',%20'.ad-slot',%20'.antenna',%20'.xrail-content',%20%20%20%20'.advertisement__leaderboard',%20'.ad-leaderboard',%20'.trc_rbox_outer',%20'.ks-recommended',%20%20%20%20'.article-da',%20'div.sponsored-stories-component',%20'div.addthis-smartlayers',%20%20%20%20'div.article-adsponsor',%20'div.signin-prompt',%20'div.article-bumper',%20'div.video-placeholder',%20%20%20%20'div.top-ad-container',%20'div.header-ad',%20'div.ad-unit',%20'div.demo-block',%20'div.OUTBRAIN',%20%20%20%20'div.ob-widget',%20'div.nwsrm-wrapper',%20'div.announcementBar',%20'div.partner-resources-block',%20%20%20%20'div.arrow-down',%20'div.m-ad',%20'div.story-interrupt',%20'div.taboola-recommended',%20%20%20%20'div.ad-cluster-container',%20'div.ctx-sidebar',%20'div.incognito-modal',%20'.OUTBRAIN',%20'.subscribe-button',%20%20%20%20'.ads9',%20'.leaderboards',%20'.GoogleActiveViewElement',%20'.mpu-container',%20'.ad-300x600',%20'.tf-ad-block',%20%20%20%20'.sidebar-ads-holder-top',%20'.ads-one',%20'.FullPageModal__scroller',%20%20%20%20'.content-ads-holder',%20'.widget-area',%20'.social-buttons',%20'.ac-player-ph',%20%20%20%20/*%20Other:%20*/%20%20%20%20'script',%20'sponsored',%20'video',%20'aside#sponsored-recommendations',%20'aside[role=%22banner%22]',%20'aside',%20%20%20%20'amp-ad',%20'span[id^=ad_is_]',%20'div[class*=%22indianapolis-optin%22]',%20'div[id^=google_ads_iframe]',%20%20%20%20'div[data-google-query-id]',%20'section[data-response]',%20'ins.adsbygoogle',%20'div[data-google-query-id]',%20%20%20%20'div[data-test-id=%22fullPageSignupModal%22]',%20'div[data-test-id=%22giftWrap%22]'%20];%20%20%20%20for(let%20i%20in%20selectors)%20{%20%20%20%20%20%20%20%20let%20nodesList%20=%20document.querySelectorAll(selectors[i]);%20%20%20%20%20%20%20%20for(let%20i%20=%200;%20i%20%3C%20nodesList.length;%20i++)%20{%20%20%20%20%20%20%20%20%20%20%20%20let%20el%20=%20nodesList[i];%20%20%20%20%20%20%20%20%20%20%20%20if(el%20&&%20el.parentNode)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20el.parentNode.removeChild(el);%20%20%20%20%20%20%20%20}%20%20%20%20}})();`

**Reply by Sxri** [reply]
Yeah i've tried that lol, it works sorta, but not that useful considering our computers also block sites on their wifi. its useful if you use a mobile hotspot from your phone only sadly.

Thanks for the TIP though! I'll make sure to use it when I have enough data to use hotspot.

**Reply by ghost** [reply]
Ok, you're welcome

**Comment by 3kh0** [comment]
There are tons of exploits in bypassi's discord (private RN) and the holy unblocker discord (discord.gg/unblock)

**Reply by Sxri** [reply]
I'm in holy unblocker discord already , when will bypassi's discord be public, do you know?

**Reply by 3kh0** [reply]
idk

**Reply by Aka-but-nice** [reply]
Refer to #348  or #347
